L4T R36.4.3

madisongh opened the pull request.

dwalkes commented:
Test sheet at https://docs.google.com/spreadsheets/d/1EWG5QZvtWdk74gz48D4xCbegK-bsqMFJ8XOsyGdUUeI/edit?gid=1760631514#gid=1760631514

loblik commented:
Is there a plan to also update scarthgap LTS to 36.4.3?

dwalkes commented:
> Is there a plan to also update scarthgap LTS to 36.4.3?

We haven't discussed yet AFAIK but I assume all the branches on 36.4 in https://github.com/OE4T/meta-tegra/wiki/Which-branch-should-I-use%3F will likely get this backport when it's ready, including scarthgap.

ichergui commented:
> Is there a plan to also update scarthgap LTS to 36.4.3?

Hi @loblik 
We need to do more testing and fix a couple of things before merging the changes.
It will be great if you can help on the testing and report any issue.
Thanks

dwalkes commented:
Went through several tests of `demo-image-full` on `jetson-orin-nano-devkit-nvme` today using https://github.com/OE4T/meta-tegra/pull/1872 just before the rebase Matt did this morning and using `linux-jammy-nvidia` kernel to avoid the issues also present on the master branch as [discussed in matrix](https://matrix.to/#/!mKaltPMAHUgQwPUKcH:gitter.im/$oGzaz97fSzDlklXrlteT_VYLRErpXab-eVOIrZ4bMJU?via=gitter.im&via=matrix.org&via=3dvisionlabs.com)

* I'm not sure if the stereo disparity tests are expected to fail on nano or not.
* I think the deepstream docker test failure is probably a missing dependency on a dev package which isn't documented.
* I'm guessing the `run-deepstream-tests` failures for python tests are probably real, haven't looked into those yet.

dwalkes commented:
6.12 issues with display driver and deviceQuery should be resolved after the work in progress for 6.12 changes on master - verified with a test rebase at https://github.com/Trellis-Logic/meta-tegra/tree/wip-l4t-r36.4.3%2Brebase-6.12-fixes

madisongh commented:
I've run some tests on an AGX Orin devkit and updated the spreadsheet. Looking good enough to merge now, so I've switched this out of draft state and will merge it in the next day or two.

dwalkes commented:
> I'm guessing the run-deepstream-tests failures for python tests are probably real, haven't looked into those yet.

I can reproduce with `python3 -c "from gi.repository import Gst; print(Gst.init)"`

Looks like they are https://gitlab.gnome.org/GNOME/pygobject/-/merge_requests/417 - will be in the next release of python3-pygobject after 3.52.3

madisongh commented:
> Looks like they are https://gitlab.gnome.org/GNOME/pygobject/-/merge_requests/417 - will be in the next release of python3-pygobject after 3.52.3

Actually, that commit [is](https://gitlab.gnome.org/GNOME/pygobject/-/commits/3.52.3) in 3.52.3, and OE-Core master updated to that version earlier this month. That explains why the DeepStream tests passed for me when I ran them yesterday before merging the PR.
